pkg/prometheus/operator.go

```diff
@@ -2157,3 +2163,19 @@ func validateRelabelConfig(rc monitoringv1.RelabelConfig) error {
 	}
 	return nil
 }
+
+func validateProberURL(url string) error {
+	hostPort := strings.Split(url, ":")
```

Comment:
https://pkg.go.dev/net#SplitHostPort might work here? Not sure if it will negate the need for the dependency on `govalidator`

Reply:
I had looked at that but we can specify in combinations of host/ip/host:port/ip:port it wont work
https://go.dev/play/p/HsReNGKhoqX